app/views/content/life-as-a-teacher/teaching-as-a-career/from-teaching-intern-to-leadership-roles/_article.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+      <h2>Becoming a maths teacher</h2>
+      <p>While I was doing my physics degree, I looked into various internships and other opportunities but none of them really appealed to me. By chance, I saw a poster advertising a physics teaching internship and thought I’d give it a go. As I was filling out the application form, I realised I had a lot of the skills they were looking for.</p>
+      <p>I wasn’t always sure what career I’d like to pursue. A physics degree appealed to me because the questions physics deals with seem so much bigger and more open than in other subjects. I had very knowledgeable physics teachers who shared their passion for the subject beyond the syllabus, and that’s something I try to do with my students now.</p>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "One of the great things about teaching is that pupils are often asking questions that I don’t know the answer to. I find their perspectives so interesting and challenging.",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+      <p>I like to bring the research I did during my degree into the classroom, and when students ask me why I decided to teach rather than pursue research I tell them that teaching is so much more interesting and fun!</p>
+      <p>My teacher training year was definitely a learning curve. I have vivid memories of my first lesson, where a task that I had thought would take five minutes ended up lasting the entire hour! I found the training really interesting because as well as practising the skills of teaching, you’re also learning the theory behind why certain approaches work. It’s what I had felt was missing from my research work — in teaching I really got to see the application of what I was doing. More recently I have continued to pursue this through an MA in Science Education at the UCL Institute of Education.</p>
+      <h2>Starting my career progression</h2>
+      <p>I applied for the Head of Physics role during my first year of teaching and was successful. This was a big jump for me and essentially allowed me to have a bigger impact beyond the walls of my classroom, improving provision for all students studying key stage 5 physics and the staff that taught them.</p>
+      <h2>Increased responsibility</h2>
+      <p>In my third year I was promoted to second in department and took on more responsibility. I lead meetings, manage the budget, conduct appraisals, and organise ‘learning walks’ to ensure teaching is consistent across the department. This extra work is reflected in my teaching schedule, with reduced classroom hours so that I can manage all the additional tasks.</p>
```

Comment:
```suggestion
      <p>In my third year I was promoted to second in department and took on more responsibility. I lead meetings, manage the budget, conduct appraisals, and organise learning walks to ensure teaching is consistent across the department. This extra work is reflected in my teaching schedule, with reduced classroom hours so that I can manage all the additional tasks.</p>
```
Do we need the quote marks?